src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -371,14 +371,39 @@ public void getProjectSitePages(@PathVariable String site_id,
 	}
 
 	/**
+	 * API call to get CTools site membership
+	 * 
+	 * @return
+	 */
+	@GET
+	@RequestMapping("/siteMembership/{siteId}")
```

Comment:
Just for your consideration, IMHO each level of the URL would produce something useful.  For /siteMembership it could return a list of possible sites (not practical or necessary in this case) or it could return a representation of the sub urls that documents how to use it.  E.g.  a sample URL that shows how to construct a real one.  Say "https://yourGoodSite.con/siteMembership/{siteId}"  This still ought to be in same (json) format so reading it is trivial.  This can make an API much easier to use since someone making a top-level request can find out what the API can do just by making requests of the API.